### DISCUSSION TITLE
Is there some way to modify the naming of the completed folders? (Adding the OMDB/TMDB ID to the end?)

### DISCUSSION THREAD

**Discussion by josh-hemphill** [automatic-ripping-machine/automatic-ripping-machine#1188] [Q&A]
I would really like to have absolute resolution after importing to my server, and Jellyfin supports tagging with the DB IDs so there's aren't issues with string searches.

**Comment by microtechno9000** [comment]
In what format?

**Reply by josh-hemphill** [reply]
e.g. `Dune (2021) [tmdbid-438631]` [(jellyfin docs)](https://jellyfin.org/docs/general/server/media/movies/#movie-extras:~:text=TIP,2018\)%20%5Btmdbid%2D65567%5D)

**Reply by josh-hemphill** [reply]
Taking another look at Jellyfin naming, it also allows for adding tags at the end to specify the resolution as version or tag it as certain cut, which would also be useful to automate.
